### PR TITLE
Add oslo.policy checks

### PR DESCRIPTION
**Pull request by tzumainn**
This PR adds oslo.policy checks, enabling alternative usages where
a single admin has administrative rights over all nodes.

Addresses #150

### CODE REVIEW

File: flocx_market/api/bid.py [hunk 1]
```diff
@@ -1,25 +1,31 @@
 from flask_restful import Resource
 from flask import request, g
```
Comment (larsks) [inline flocx_market/api/bid.py]: Really, flask? You're going to name a public API attribute `g`?

File: flocx_market/common/policy.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+CONF = flocx_market.conf.CONF
+_ENFORCER = None
+
+default_policies = [
+    policy.RuleDefault('is_admin',
+                       'role:admin or role:flocx_market_admin',
+                       description='Full API access'),
+]
+
+bid_policies = [
```
Comment (larsks) [inline flocx_market/common/policy.py]: If I'm reading this properly, the default policies are all admin-only, right? The idea being that an operator would always need to deploy a customized policy file to make the service generally available?

File: flocx_market/api/bid.py (hunk 1 above)
Comment (tzumainn) [reply flocx_market/api/bid.py]: I had a very similar reaction :)

File: flocx_market/common/policy.py (hunk 2 above)
Comment (tzumainn) [reply flocx_market/common/policy.py]: Yes. I have decided this is the best way to preserve backwards compatibility with everyone using this code.

Comment (larsks) [reply flocx_market/common/policy.py]: I don't think there *is* anyone using this code :)

Comment (tzumainn) [reply flocx_market/common/policy.py]: what, no, I'm sure there are billions

**Pull request merged**